test/cargo_build_script/build.rs

```diff
@@ -18,9 +18,21 @@ fn test_encoded_rustflags() {
     assert_eq!(flags[1], "--verbose");
 }
 
+
+/// Ensure Make variables provided by the `toolchains` attribute are expandable.
+fn test_toolchain_var() {
+    let tool_path = std::env::var("EXPANDED_TOOLCHAIN_VAR").unwrap();
```

Comment:
Let's also assert that the path pointed to exists?

Reply:
Done!